code/src/components/ToDoList.js

```diff
@@ -0,0 +1,242 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import styled, { keyframes } from "styled-components";
+
+import tasks, { selectFilteredTodos } from "../reducers/tasks";
+import moment from "moment";
+
+const ToDoList = () => {
+  const items = useSelector((store) => selectFilteredTodos(store));
+
+  const dispatch = useDispatch();
+
+  const onToggleTask = (id) => {
+    dispatch(tasks.actions.toggleTask(id));
+  };
+
+  const onClickCompleteAll = () => {
+    dispatch(tasks.actions.completeAllTodos());
+  };
+
+  const onDeleteTask = (id) => {
+    dispatch(tasks.actions.deleteTask(id));
+  };
+
+  return (
+    <ToDoContainer>
+      {items.map((item) => (
+        <div key={item.id}>
+          <TasksContainer>
+            <InfoContainer>
+              <Task>{item.text}</Task>
+              <Date>{moment().format("L")}</Date>
+            </InfoContainer>
+            <IconsContainer>
+              <Label>
+                <Input
+                  type="checkbox"
+                  checked={item.isComplete}
+                  onChange={() => onToggleTask(item.id)}
+                />
+                <Indicator />
+              </Label>
+
+              <DeleteButton onClick={() => onDeleteTask(item.id)}>
+                <i className="fas fa-times-circle fa-lg"></i>
+              </DeleteButton>
+            </IconsContainer>
+          </TasksContainer>
+        </div>
+      ))}
+      <ButtonsContainer>
+        <CompleteButton
+          disabled={items.length < 1}
```

Comment:
Is it really necessary to disable the button since you remove the button when the length of the list is 0?